src/JuliaFormatter.jl

```diff
@@ -633,6 +633,10 @@ function format_text(cst::CSTParser.EXPR, style::AbstractStyle, s::State)
 
     needs_alignment(s.opts) && align_fst!(fst, s.opts)
 
+    if s.opts.long_to_short_function_def && s.opts.short_to_long_function_def
+        @warn "Both `long_to_short_function_def` and `short_to_long_function_def` are set"
+    end
+
```

Comment:
this should be done as earlier as possible in the process so ideally before we we initially parse the code

Reply:
Is the new position in https://github.com/domluna/JuliaFormatter.jl/pull/629/commits/d091dd1690b4ca755c25d0881112687f22ff5f16 ok ?

Reply:
yeah that looks good